PULL-EXAMPLE.md

```diff
@@ -0,0 +1,114 @@
+# Pull Request Example #
+
+Thank you for contributing to CISA's Log4j-affected-db GitHub Repository! Please
+follow the steps listed below in order to add a product to the public
+repository. **Note:** To assure the accuracy of data please only include
+products that have official advisories or alerts that verify the product's
+vulnerability status to  [CVE-2021-4104](https://nvd.nist.gov/vuln/detail/cve-2021-4104),
+[CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228),
+[CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046),
+and/or [CVE-2021-45105](https://nvd.nist.gov/vuln/detail/CVE-2021-45105?s=09).
+
+**Step 1:** Go to the cisagov data .yml files and choose the appropriate file to
+include your updates (example link). The files are separated alphabetically
+based on the first letter of the vendor name. For example, 'CISA' would be
+located in [`cisagov_C.yml`](https://github.com/cisagov/data/cisagov_C.yml).
```

Comment:
Here are some suggestions for this block:
* Add a link to the data directory (won't work right now, but will be correct after this PR is merged)
* Remove the "example link" text
* Correct the `cisagov_C.yml` URL (won't work right now, but will be correct after this PR is merged)
```suggestion
**Step 1:** Go to the [cisagov data .yml files](https://github.com/cisagov/log4j-affected-db/tree/develop/data/)
and choose the appropriate file to include your updates. 
The files are separated alphabetically based on the first letter of the
vendor name. For example, 'CISA' would be located in 
[`cisagov_C.yml`](https://github.com/cisagov/log4j-affected-db/blob/develop/data/cisagov_C.yml).
```

Reply:
@Lcerkov - sorry about that trailing whitespace in my suggestion above.  Thanks for fixing it in https://github.com/cisagov/log4j-affected-db/pull/425/commits/38fe6123ee712b7f35d3d31e014c7b0e6431767b. 👍